doc/polyscope_compatibility.rst

```diff
@@ -1,27 +1,41 @@
-Polyscope version compatibility
-===============================
+|polyscope| version compatibility
+=================================
 
-The table below shows breaking changes in the library compared to Polyscope versions. Compatibility
+The table below shows breaking changes in the library compared to |polyscope| versions. Compatibility
 is listed for CB3 robots (versions 3.x.y) and e-Series robots (versions 5.x.y) respectively.
 
 It might be possible to use the library in a more recent version than the compatible versions listed
 below, but that might either require some manual modifications or not using a subset of features.
 For that reason the breaking changes are also listed inside the table. Please note that choosing a
 higher library version than recommended happens on your own risk and will not be supported.
 
-If your Polyscope version is less than the minimum required version for the latest library version,
+If your |polyscope| version is less than the minimum required version for the latest library version,
 we suggest to upgrade your robot's software. Please refer to the robot's user manual how to update
 your robot.
 
 .. list-table::
    :header-rows: 1
 
-   * - Polyscope version
-     - Max. version
-     - Max. tag
+   * - |polyscope| version
+     - Version
+     - Tag
      - Breaking changes
-   * - < 3.14.3 / 5.9.4
+   * - < 3.12.3 / 5.5.1
      - 1.1.0
      - `polyscope_compat_break_1 <https://github.com/UniversalRobots/Universal_Robots_Client_Library/tree/polyscope_compat_break_1>`_
      - `tcp_offset in RTDE interface <https://github.com/UniversalRobots/Universal_Robots_Client_Library/pull/110>`_
+   * - < 3.14.3 / 5.9.4
+     - 1.3.2
```

Comment:
```suggestion
     - 1.3.1
```
#151 was merged before 1.3.2. so the maximum release version that can be used, should be 1.3.1. I've created the `polyscope_compat_break_2` tag on the commit before merging in #151.